### ISSUE TITLE
The accuracy of the pre-trained model is different from the accuracy reported by pytorch

### ISSUE THREAD

**Issue by JingLiu95** [NVIDIA/DALI#400]
I run the example in docs/examples/pytorch/resnet50. Typing following command
`CUDA_VISIBLE_DEVICES=0,1,2,3 python -m torch.distributed.launch --nproc_per_node=4 main.py --b 100 --pretrained --evaluate /home/dataset/imagenet`.
The accuracy of the pre-trained resnet18 is 69.024. However, the accuracy reported by pytorch official is 69.76. 

DALI 0.6
python 2.7
pytorch 1.0

**Comment by JanuszL** [comment]
Hi,
It may be related to #389. Could you check how it works without DALI using APEX example: https://github.com/NVIDIA/apex/tree/master/examples/imagenet?

**Comment by JanuszL** [comment]
Tracked as DALI-473

**Comment by JanuszL** [comment]
Hi,
The thing you observes is a result of discrepancies between DALI and PyTorch data processing pipeline. PyTorch uses Pillow while DALI for CPU uses OpenCV and npp for GPU. The same operations don't return bit-exact results. Recently we moved to a custom implementation of the resize with resampling:
* #520
* #652
* #642
It will be available in DALI 0.8. I tested the command you run with new resize implementation and DALI gives me:
```
Prec@1 69.828 Prec@5 89.078
```
Numbers are even better than PyTorch official.
Updated example is under review #663.

**Comment by JanuszL** [comment]
I have forgotten to mention that if you train and validate with DALI then you will get 69.76 or better accuracy. The problem is more visible when your training and validation pipelines differ by any mean (doesn't return bit exact values - no matter if this is JPEG decoding or augmentations).

**Comment by JanuszL** [comment]
0.8 is available [here](https://github.com/NVIDIA/DALI/releases/tag/v0.8.0)
